python/ray/setup-dev.py

```diff
@@ -51,16 +58,19 @@ def do_link(package, force=False, local_path=""):
         "--yes", "-y", action="store_true", help="Don't ask for confirmation.")
     args = parser.parse_args()
 
-    do_link("rllib", force=args.yes, local_path="../../")
+    do_link("rllib", force=args.yes, local_path="../../../rllib")
     do_link("tune", force=args.yes)
     do_link("autoscaler", force=args.yes)
     do_link("scripts", force=args.yes)
     do_link("internal", force=args.yes)
     do_link("tests", force=args.yes)
     do_link("experimental", force=args.yes)
     do_link("util", force=args.yes)
-    do_link("dashboard", force=args.yes)
-    do_link("new_dashboard", force=args.yes)
+    do_link("dashboard", force=args.yes, local_path="../../../dashboard")
```

Comment:
You can remove this line altogether. We don't actually require this symlink anymore. We start the new dashboard by default, and there isn't a way of running the old dashboard code anymore. We've been taking parts out.

Reply:
Thanks for clarifying @mfitton .